src/content/documentation/develop/getting-started-with-web-ext.md

```diff
@@ -85,7 +86,7 @@ web-ext --version
 
 ## Using web-ext
 
-Before you start using `web-ext` locate an example extension to use—if you don’t have one, use one from the [webextensions-examples](https://github.com/mdn/webextensions-examples) repo.
+Before you start using `web-ext` locate an example extension to use—if you don’t have one, use one from the [webextensions-examples](https://github.com/mdn/webextensions-examples) repo. If you would like to start fresh, use our [boilerplating tool](https://extensionworkshop.com/documentation/develop/browser-extension-development-tools/#boilerplating-tools) to get started with a fresh extension.
```

Comment:
Small wording change: Let's start the second sentence with, "If you would like to start from scratch,". Otherwise, this looks good!

Reply:
@caitmuenster given that the authors of the boilerplate tool linked in that page are our contributors Shivam and Trishul, how about changing `our boilerplating tool` with `one of the boilerplating tools` or something like that (just to don't steal the credits ;-), I think that "our" would be more likely read as "released by Mozilla" in this context), wdyt?

Reply:
Oh, I hadn't noticed that. Thank you @rpl mentioning this. I will update it as soon as we what @caitmuenster thinks about it.

Reply:
We may as well go one step further and change `our` to `our community developed boilerplating tool`. What do you think @rpl, @caitmuenster ?

Reply:
I leave the final choice (and feedback comments) of the exact wording to @caitmuenster (especially given I'm not native in english ;-))

Reply:
Good thought @rpl! 

@ankushduacodes, I like your suggestion to change `our` to `our community developed boilerplating tool`.